### ISSUE TITLE
Running yaml-resume after installing fails, classic.css and requirements are missing

### ISSUE THREAD

**Issue by signalkraft** [notsag/yaml-resume#25]
**Describe the bug**
Following the docs on latest Ubuntu doesn't work, necessary requirements aren't found after running `pip3 install yaml-resume`:

```
Collecting yaml-resume
  Downloading yaml-resume-0.7.9.tar.gz (24 kB)
Collecting click
  Downloading click-7.1.2-py2.py3-none-any.whl (82 kB)
Collecting pyyaml
  Downloading PyYAML-5.4.1-cp39-cp39-manylinux1_x86_64.whl (630 kB)
Collecting cerberus
  Downloading Cerberus-1.3.4.tar.gz (63 kB)
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'done'
    Preparing wheel metadata: started
    Preparing wheel metadata: finished with status 'done'
Requirement already satisfied: setuptools in /opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages (from cerberus->yaml-resume) (56.0.0)
Using legacy 'setup.py install' for yaml-resume, since package 'wheel' is not installed.
Building wheels for collected packages: cerberus
  Building wheel for cerberus (PEP 517): started
  Building wheel for cerberus (PEP 517): finished with status 'done'
  Created wheel for cerberus: filename=Cerberus-1.3.4-py3-none-any.whl size=58175 sha256=2a1f5d1edeae2f97bd787814548fc76a0b3ed89fdad40edb6b195f50203973b4
  Stored in directory: /home/runner/.cache/pip/wheels/52/26/88/27d9b76e7756eedcf3d6e8fcc3bc28da0869749ffe8d87f7ac
Successfully built cerberus
Installing collected packages: pyyaml, click, cerberus, yaml-resume
    Running setup.py install for yaml-resume: started
    Running setup.py install for yaml-resume: finished with status 'done'
Successfully installed cerberus-1.3.4 click-7.1.2 pyyaml-5.4.1 yaml-resume-0.7.9
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.5/x64/bin/yaml-resume", line 33, in <module>
    sys.exit(load_entry_point('yaml-resume==0.7.9', 'console_scripts', 'yaml-resume')())
  File "/opt/hostedtoolcache/Python/3.9.5/x64/bin/yaml-resume", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/importlib/metadata.py", line 77, in load
    module = import_module(match.group('module'))
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/yaml_resume/cli.py", line 4, in <module>
    from jinja2 import Environment, PackageLoader
ModuleNotFoundError: No module named 'jinja2'
```

...and when I install them from your requirements file, the `classic.css` file isn't found because it doesn't seem to be included in the PyPi package:

```
Collecting click>=7.0
  Downloading click-7.1.2-py2.py3-none-any.whl (82 kB)
Collecting pyyaml>=5.0
  Downloading PyYAML-5.4.1-cp39-cp39-manylinux1_x86_64.whl (630 kB)
Collecting cerberus>=1.0
  Downloading Cerberus-1.3.4.tar.gz (63 kB)
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'done'
    Preparing wheel metadata: started
    Preparing wheel metadata: finished with status 'done'
Collecting jinja2>=2.10
  Downloading Jinja2-2.11.3-py2.py3-none-any.whl (125 kB)
Collecting WeasyPrint==52.1
  Downloading WeasyPrint-52.1-py3-none-any.whl (304 kB)
Collecting yaml-resume==0.7.9
  Downloading yaml-resume-0.7.9.tar.gz (24 kB)
Collecting tinycss2>=1.0.0
  Downloading tinycss2-1.1.0-py3-none-any.whl (21 kB)
Collecting html5lib>=0.999999999
  Downloading html5lib-1.1-py2.py3-none-any.whl (112 kB)
Requirement already satisfied: setuptools>=39.2.0 in /opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages (from WeasyPrint==52.1->-r requirements.txt (line 5)) (56.0.0)
Collecting cssselect2>=0.1
  Downloading cssselect2-0.4.1-py3-none-any.whl (13 kB)
Collecting cffi>=0.6
  Downloading cffi-1.14.5-cp39-cp39-manylinux1_x86_64.whl (406 kB)
Collecting CairoSVG>=2.4.0
  Downloading CairoSVG-2.5.2-py3-none-any.whl (45 kB)
Collecting Pyphen>=0.9.1
  Downloading Pyphen-0.10.0-py3-none-any.whl (1.9 MB)
Collecting Pillow>=4.0.0
  Downloading Pillow-8.2.0-cp39-cp39-manylinux1_x86_64.whl (3.0 MB)
Collecting cairocffi>=0.9.0
  Downloading cairocffi-1.2.0.tar.gz (70 kB)
Collecting MarkupSafe>=0.23
  Downloading MarkupSafe-1.1.1-cp39-cp39-manylinux2010_x86_64.whl (32 kB)
Collecting defusedxml
  Downloading defusedxml-0.7.1-py2.py3-none-any.whl (25 kB)
Collecting pycparser
  Using cached pycparser-2.20-py2.py3-none-any.whl (112 kB)
Collecting webencodings
  Downloading webencodings-0.5.1-py2.py3-none-any.whl (11 kB)
Collecting six>=1.9
  Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
Using legacy 'setup.py install' for yaml-resume, since package 'wheel' is not installed.
Using legacy 'setup.py install' for cairocffi, since package 'wheel' is not installed.
Building wheels for collected packages: cerberus
  Building wheel for cerberus (PEP 517): started
  Building wheel for cerberus (PEP 517): finished with status 'done'
  Created wheel for cerberus: filename=Cerberus-1.3.4-py3-none-any.whl size=58175 sha256=51369faf2b19faa6ab876c50e910c8f72555a03234e289f45b8d003e35123c76
  Stored in directory: /home/runner/.cache/pip/wheels/52/26/88/27d9b76e7756eedcf3d6e8fcc3bc28da0869749ffe8d87f7ac
Successfully built cerberus
Installing collected packages: webencodings, pycparser, tinycss2, cffi, six, Pillow, defusedxml, cssselect2, cairocffi, pyyaml, Pyphen, MarkupSafe, html5lib, click, cerberus, CairoSVG, yaml-resume, WeasyPrint, jinja2
    Running setup.py install for cairocffi: started
    Running setup.py install for cairocffi: finished with status 'done'
    Running setup.py install for yaml-resume: started
    Running setup.py install for yaml-resume: finished with status 'done'
Successfully installed CairoSVG-2.5.2 MarkupSafe-1.1.1 Pillow-8.2.0 Pyphen-0.10.0 WeasyPrint-52.1 cairocffi-1.2.0 cerberus-1.3.4 cffi-1.14.5 click-7.1.2 cssselect2-0.4.1 defusedxml-0.7.1 html5lib-1.1 jinja2-2.11.3 pycparser-2.20 pyyaml-5.4.1 six-1.16.0 tinycss2-1.1.0 webencodings-0.5.1 yaml-resume-0.7.9
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.5/x64/bin/yaml-resume", line 33, in <module>
    sys.exit(load_entry_point('yaml-resume==0.7.9', 'console_scripts', 'yaml-resume')())
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/yaml_resume/cli.py", line 148, in export
    html = HTML(string=template.render(resume=resume, image=image))
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/yaml_resume/templates/classic.html", line 5, in top-level template code
    <style type="text/css">{% include "classic.css" %}</style>
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/jinja2/loaders.py", line 249, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: classic.css
```

I tried directly installing from git with `pip3 install git+https://github.com/notsag/yaml-resume.git` but that didn't fix the issue, so I'm guessing something in the packaging is causing this?

Same issues happened on macOS as well.

**To Reproduce**
1. Follow official docs on a "blank" Ubuntu with Python3

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Version: 18.04 and latest
 - Python Version: 3.9

**Comment by notsag** [comment]
Hello @signalkraft,

Python 3.9 is not officially supported as I haven't setup CI for it yet.
Still, I manually tested on a ubuntu20.04 docker container and it seems to be working for me :
```
root@5f3273391479:/# python3.9 -m pip install git+https://github.com/notsag/yaml-resume.git
Collecting git+https://github.com/notsag/yaml-resume.git
  Cloning https://github.com/notsag/yaml-resume.git to /tmp/pip-req-build-glo1zutc
  Running command git clone -q https://github.com/notsag/yaml-resume.git /tmp/pip-req-build-glo1zutc
Collecting click
  Downloading click-8.0.0-py3-none-any.whl (96 kB)
     |████████████████████████████████| 96 kB 2.6 MB/s 
Collecting pyyaml
  Downloading PyYAML-5.4.1-cp39-cp39-manylinux1_x86_64.whl (630 kB)
     |████████████████████████████████| 630 kB 6.0 MB/s 
Collecting cerberus
  Downloading Cerberus-1.3.4.tar.gz (63 kB)
     |████████████████████████████████| 63 kB 3.0 MB/s 
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Requirement already satisfied: setuptools in /root/.local/lib/python3.9/site-packages (from cerberus->yaml-resume==0.7.9) (56.2.0)
Building wheels for collected packages: yaml-resume, cerberus
  Building wheel for yaml-resume (setup.py) ... done
  Created wheel for yaml-resume: filename=yaml_resume-0.7.9-py3-none-any.whl size=28948 sha256=b9980d7dc079c269b1b88587ab839bde6ab4c799547a61e4626d8ede2fef15fa
  Stored in directory: /tmp/pip-ephem-wheel-cache-12p2rxw3/wheels/a9/e1/3f/09e8f5447958fb68604ddbe8e660e3a79f11e8fc22dfb9bcb4
  Building wheel for cerberus (PEP 517) ... done
  Created wheel for cerberus: filename=Cerberus-1.3.4-py3-none-any.whl size=58175 sha256=bbe1bfc69330383c51fac864b92af0375a973ddaf8fabd74848eefa8bfbce5b6
  Stored in directory: /root/.cache/pip/wheels/52/26/88/27d9b76e7756eedcf3d6e8fcc3bc28da0869749ffe8d87f7ac
Successfully built yaml-resume cerberus
Installing collected packages: pyyaml, click, cerberus, yaml-resume
Successfully installed cerberus-1.3.4 click-8.0.0 pyyaml-5.4.1 yaml-resume-0.7.9
```

The first error seems to be related to the installation of the `jinja2` package. Can you try manually installing dependancies using :
```
pip3 install -r requirements.txt
```

And then try running the pip install from pypi.

**Comment by iwikmai** [comment]
Same issue here with MacOS Python3.9

`❯ yaml-resume export ~/Downloads/resume_19.yml -e pdf -o ~/Downloads/Resume.pdf
Traceback (most recent call last):
  File "/usr/local/bin/yaml-resume", line 8, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/yaml_resume/cli.py", line 148, in export
    html = HTML(string=template.render(resume=resume, image=image))
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/usr/local/lib/python3.9/site-packages/yaml_resume/templates/classic.html", line 5, in top-level template code
    <style type="text/css">{% include "classic.css" %}</style>
  File "/usr/local/lib/python3.9/site-packages/jinja2/loaders.py", line 325, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: classic.css`

**Comment by notsag** [comment]
@iwikmai how did you install yaml-resume? What version are you running?

**Comment by notsag** [comment]
Ok I noticed I wasn't including the CSS files in the distribution.
I fixed this on the master branch. I still need to fix some CI issues before releasing a new version on pypi.

**Comment by notsag** [comment]
This is now fixed in 0.7.10.
